tests/Dfc.CourseDirectory.Web.Tests/LayoutModelStorageTests.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using Dfc.CourseDirectory.Web.Views.Shared;
+using FluentAssertions;
```

Comment:
We don't use FluentAssertions anywhere else currently AFAIK. For me we should use xUnit straight until we've committed to FluentAssertions everywhere.

Reply:
stripped out and forcepushed :+1: